### PR TITLE
Use ChatInfo instead of ChatMessage when target is not found

### PR DESCRIPTION
**Pull request by WoozChucky**
GiveItem command was incorrectly sending a normal chat message when the Target player is not found.

Now it sends a ChatInfo message visible only to the player.

### CODE REVIEW

File: src/Tests/Core.Tests/CommandTests.cs [hunk 1]
```diff
@@ -286,6 +286,17 @@ public async Task GiveOtherItemCommand()
             Count = 10
         }, cfg => cfg.Including(x => x.ItemId).Including(x => x.Count));
     }
+    
+    [Fact]
+    public async Task GiveItemCommand_InvalidPlayer()
+    {
+        await _commandManager.Handle(_connection, "/give missing 1 10");
+
+        (_connection as MockedGameConnection).SentMessages.Should().ContainEquivalentOf(new ChatOutcoming
```
Comment (MeikelLP) [inline src/Tests/Core.Tests/CommandTests.cs:295]: Use a direct cast here to avoid warnings :)

**Review by MeikelLP — CHANGES_REQUESTED** [review]

File: src/Tests/Core.Tests/CommandTests.cs (hunk 1 above)
Comment (MeikelLP) [reply src/Tests/Core.Tests/CommandTests.cs:295]: No. Like this:

```cs
((MockedGameConnection)_connection).SentMessages.Should().ContainEquivalentOf(...)
```

Comment (WoozChucky) [reply src/Tests/Core.Tests/CommandTests.cs]: Thanks for the explanation! 😄  It's been a while since I last used direct casting as such

Comment (MeikelLP) [reply src/Tests/Core.Tests/CommandTests.cs]: Its not like the old is obsolete or anything. They serve different purposes :)

**Review by MeikelLP — APPROVED** [review]

**Comment by MeikelLP** [comment]
FYI: I noticed that the CI pipeline for sonar is failing. I already found a possible solution but I can't apply it right now. I will update u asap

**Comment by MeikelLP** [comment]
Please merge master into your fork and try again. Same with your other PR

**Comment by WoozChucky** [comment]
> Please merge master into your fork and try again. Same with your other PR

Done 👍

**Pull request merged**